server/src/main/java/org/apache/iotdb/db/engine/storagegroup/timeindex/TimeIndexLevel.java

```diff
@@ -20,14 +20,19 @@
 package org.apache.iotdb.db.engine.storagegroup.timeindex;
 
 public enum TimeIndexLevel {
-  /** file to time index (small memory foot print) */
-  FILE_TIME_INDEX,
+  /** old file to time index (small memory foot print) */
+  OLD_FILE_TIME_INDEX,
```

Comment:
Better rename to V012_FILE_TIME_INDEX, make it more specific.

Reply:
fixed